command/plugins_remove.go

```diff
@@ -46,11 +57,75 @@ func (c *PluginsRemoveCommand) Run(args []string) int {
 	return c.RunContext(ctx, args)
 }
 
+// deletePluginBinary removes a local plugin binary, and its related checksum file.
+func deletePluginBinary(pluginPath string) error {
+	if err := os.Remove(pluginPath); err != nil {
+		return err
+	}
+	shasumFile := fmt.Sprintf("%s_SHA256SUM", pluginPath)
+
+	if _, err := os.Stat(shasumFile); err != nil {
+		log.Printf("[INFO] No SHA256SUM file to remove for the plugin, ignoring.")
+		return nil
+	}
+
+	return os.Remove(shasumFile)
+}
+
 func (c *PluginsRemoveCommand) RunContext(buildCtx context.Context, args []string) int {
 	if len(args) < 1 || len(args) > 2 {
 		return cli.RunResultHelp
 	}
 
+	pluginDir, err := packer.PluginFolder()
+	if err != nil {
+		return writeDiags(c.Ui, nil, hcl.Diagnostics{
+			&hcl.Diagnostic{
+				Severity: hcl.DiagError,
+				Summary:  "Failed to get the plugin directory",
+				Detail: fmt.Sprintf(
+					"The directory in which plugins are installed could not be fetched from the environment. This is likely a Packer bug. Error: %s",
+					err),
+			},
+		})
+	}
+
+	if filepath.IsAbs(args[0]) {
+		if len(args) != 1 {
+			c.Ui.Error("Unsupported: no version constraint may be specified with a local path.\n")
```

Comment:
```suggestion
			c.Ui.Error("Unsupported: no version constraint may be specified when specifying a local plugin path.\n")
```

Reply:
The "when specifying" feels redundant a bit, I vote in favour of leaving the "with" phrasing, but I agree about the `plugin path`, what do you think?